server/src/test/java/org/apache/cassandra/sidecar/TestModule.java

```diff
@@ -133,12 +135,15 @@ protected SidecarConfigurationImpl abstractConfig(SslConfiguration sslConfigurat
                                 .build();
         RestoreJobConfiguration restoreJobConfiguration =
         RestoreJobConfigurationImpl.builder()
-                                   .restoreJobTablesTtlSeconds(TimeUnit.DAYS.toSeconds(14) + 1)
+                                   .restoreJobTablesTtl(SecondBoundConfiguration.parse((TimeUnit.DAYS.toSeconds(14) + 1) + "s"))
```

Comment:
It'd be great to be able to sum DurationSpecs directly. But probably overengineering.

Reply:
yeah, I don't think there's a real use case for production code. Maybe a nice to have for tests, which I don't think we should take on. In any case, it's outside the scope of this PR